engine/runtime-instrument-id-execution/src/main/java/org/enso/interpreter/instrument/IdExecutionInstrument.java

```diff
@@ -234,7 +234,7 @@ public void onReturnValue(VirtualFrame frame, Object result) {
                   functionCallInstrumentationNode.getId(),
                   result,
                   nanoTimeElapsed,
-                  frame.materialize(),
+                  frame == null ? null : frame.materialize(),
```

Comment:
Kotlin `?.` safe call operator would be handy here

Reply:
Yes, it would be nice.

Reply:
Or do: `Optional.of(frame).map(f -> f.materialize()).getOrNull()` to get more functional feeling.